pkg/plugin/v3/api.go

```diff
@@ -43,6 +43,10 @@ import (
 // TODO: remove this when a better solution for using addons is implemented.
 const KbDeclarativePatternVersion = "v0.0.0-20200522144838-848d48e5b073"
 
+// AddonPlugin is the name of the plugin for addons.
+// TODO: the addon should be a plugin. More info: #1543
+const AddonPlugin = "addon"
+
```

Comment:
What does this change have to do with this PR?

Reply:
I agree with @Adirio that this cleanup would be better addressed in another PR.
Could you please revert these changes and if you wish to create a pr just with the const and the comment?

Reply:
Without this change, the golint fails that the "addon" has been used 3 time, make it a constant. Hence, I have added this change here.

Reply:
Hi @prafull01, 

I think it was falling before the reviews and the changes made since it does not fail in the master/other prs.  
I think we can revert the changes made in this file. 
Also, see that the `AddonPlugin` is not used in another place so should not be exported.

Reply:
Yeah, reverted it. @Adirio Thanks for your review.